enaml/src/colorext.cpp

```diff
@@ -220,11 +241,6 @@ struct module_state {
 };
 
 
-static PyMethodDef
-colorext_methods[] = {
```

Comment:
You can't delete this. This is the methoddef for the module, not the class. Look at line 248 in the original file. You need to rename this object to something else and update the moduledef.

Reply:
Or rather, don't delete this, and rename your methoddef object for the class to something else.

Reply:
The point of @sccolbert was, I believe, that you are mixing up the methods of the class object (that you need to add) and the ones of the module object which should not change and hence why you must not delete that piece of code.

Reply:
^ this